src/common/keybinding/index.js

```diff
@@ -0,0 +1,32 @@
+const isOsx = process.platform === 'darwin'
+
+const _normalizeAccelerator = accelerator => {
+  return accelerator.toLowerCase()
+    .replace('commandorcontrol', isOsx ? 'cmd' : 'ctrl')
+    .replace('cmdorctrl', isOsx ? 'cmd' : 'ctrl')
+    .replace('control', 'ctrl')
+    .replace('meta', 'cmd') // meta := cmd (macOS only) or super
+    .replace('command', 'cmd')
+    .replace('option', 'alt')
+}
+
+export const isEqualAccelerator = (a, b) => {
+  a = _normalizeAccelerator(a)
+  b = _normalizeAccelerator(b)
+  const i1 = a.indexOf('+')
+  const i2 = b.indexOf('+')
+  if (i1 === -1 && i2 === -1) {
+    return a === b
+  } else if (i1 === -1 || i2 === -1) {
+    return false
+  }
+
+  const partsA = a.split('+')
+  const partsB = b.split('+')
+  if (partsA.length !== partsB.length || partsA.length <= 1) {
```

Comment:
Why do you only judge `partsA.length <= `? is it necessary after you already judge `i1 === -1 || i2 === -1`?

Reply:
It's used to check whether the input is a valid Electron shortcut with a key count greater or equal to 2.